Dockerfile

```diff
@@ -98,9 +95,34 @@ RUN mkdir src &&\
     ${__wget} http://archive.apache.org/dist/xerces/c/3/sources/xerces-c-3.2.3.tar.gz |\
       ${__untar} &&\
     cmake -B src/build -S src -DCMAKE_INSTALL_PREFIX=${__prefix} &&\
-    cmake --build src/build --target install &&\
+    cmake --build src/build --target install -j$NPROC &&\
     rm -rf src
 
+###############################################################################
+# PYTHIA6
+#
+# Needed for GENIE. Needs to be linked with ROOT.
+# 
```

Comment:
Could you add an explanation of all the steps you are doing below?

If it is a standard pythia compilation, perhaps just copying in a link to the docs is enough?

Reply:
Done.